src/NServiceBus.AcceptanceTesting/AcceptanceTestingPersistence/AcceptanceTestingSynchronizedStorageSession.cs

```diff
@@ -13,7 +13,16 @@ class AcceptanceTestingSynchronizedStorageSession : ICompletableSynchronizedStor
 {
     public AcceptanceTestingTransaction Transaction { get; private set; }
 
-    public void Dispose() => Transaction = null;
+    public void Dispose()
+    {
+        Transaction = null;
+    }
+
+    public ValueTask DisposeAsync()
+    {
+        Transaction = null;
+        return ValueTask.CompletedTask;
```

Comment:
I think you can return default which represents completed since ValueTask is a struct